Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:alpine
+
+COPY src src
+WORKDIR src
+RUN npm install
+
+ENTRYPOINT ["node", "index.js"]
```

Comment:
I think you should use CMD instead of entrypoint to follow Docker best practices.

Reply:
Or if you thin this is a better way to use with the command to specify the arguments, I think it should be specified in the Readme the way to use to.

Reply:
> I think you should use CMD instead of entrypoint to follow Docker best practices.

`CMD` parameters are overwritten, when arguments are passt to `docker run`, as I propose in `run_in_docker.sh`.
Since I want both `node` and `index.js` to not be replaced, I have to use `ENTRYPOINT`, or am I wrong?
Maybe you can propose a solution, that is best practice?

> I think it should be specified in the Readme the way to use to.

Sure, I can put the content of `run_in_docker.sh` into the README.

Reply:
Moved docker instructions into [README.md](https://github.com/dorianboulch/gogs-to-gitlab/pull/4/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) with 6025c45cf1a80caf488436fa4b82ae0ec33e6a55